crates/bevy_scene/src/scene_spawner.rs

```diff
@@ -52,234 +105,210 @@ pub enum SceneSpawnError {
 }
 
 impl SceneSpawner {
-    pub fn spawn_dynamic(&mut self, scene_handle: Handle<DynamicScene>) {
-        self.dynamic_scenes_to_spawn.push(scene_handle);
+    /// Spawn a scene.
+    ///
+    /// This will only update the world when [`scene_spawner_system`] runs, see
+    /// [`SceneSpawner::spawn_sync`] for a method with immediate world update.
+    ///
+    /// The returned [`InstanceId`] can be used later to refer to the specific
+    /// instance of the scene you spawned.
+    pub fn spawn(&mut self, scene_handle: Handle<Scene>) -> InstanceId {
+        let instance_id = InstanceId::new();
+        self.scenes_to_spawn
+            .push((scene_handle.into(), instance_id));
+        instance_id
     }
 
-    pub fn spawn(&mut self, scene_handle: Handle<Scene>) -> InstanceId {
+    /// Spawn a dynamic scene. See [`SceneSpawner::spawn`].
```

Comment:
The short doc strings with reference to the relevant `Scene` implementation is to avoid repeating the same doc string over and over. Since we have here 5 methods that are basically identical with a single `DynamicScene`, deduplicating the doc helps tremendously.